### PR TITLE
EOS-13233 NFS job is failing for cortx-1.0 branch

### PR DESCRIPTION
**Pull request by SwapnilGaonkar7**
In this pull request we are adding support for space in path in which our code is present. Currently config/build is failing in path contains space in it (e.g /root/with space/temp/...).
To fix above issue I have encapsulated path uses with quotes, in some cases splited commands in 2 parts etc.

UT : refer to Seagate/cortx-posix#281

### CODE REVIEW

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by s-arya — APPROVED** [review]
LGTM. +1

File: c-utils/scripts/build.sh [hunk 1]
```diff
@@ -84,22 +84,22 @@ cortx_utils_configure() {
...
 ###############################################################################
 cortx_utils_make() {
-    if [ ! -d $CORTX_UTILS_BUILD ]; then
+    if [ ! -d "$CORTX_UTILS_BUILD" ]; then
         echo "Build folder does not exist. Please run 'config'"
         exit 1;
     fi
 
-    cd $CORTX_UTILS_BUILD
+    cd "$CORTX_UTILS_BUILD"
```
Comment (Shreya-18) [inline c-utils/scripts/build.sh:102]: Nice work!!

**Review by Shreya-18 — APPROVED** [review]
Nice work!! Looks good, +1

**Pull request merged**